src/work-allocation/components/constants/derived-icon.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const DERIVED_ICON_CONSTANTS = {
+    // current example settings for when icon should show
+    SOURCE_COLUMN: 'location',
```

Comment:
Can I suggest you use a different column for this while it's a placeholder? You've introduced an `alert` field in some of the tasks returned by the api so you could potentially make this:
```
{
  SOURCE_COLUMN: 'alert',
  MATCH_VALUE: true
}
```